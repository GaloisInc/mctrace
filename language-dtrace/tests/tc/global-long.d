long x;
unsigned long y;
long long lx;
unsigned long long ly;
