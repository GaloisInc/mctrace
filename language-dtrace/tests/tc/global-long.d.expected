TCExpected { globals = [ ("x", BVType 64)
                     , ("lx", BVType 64)
                     , ("y", BVType 64)
                     , ("ly", BVType 64)
                     ]
         }
